WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -122,10 +122,9 @@ private extension ItemListView {
         .padding(.bottom, Constants.bannerCardPadding)
     }
 
-    private var bannerHintAndLearnMoreText: Text {
-        Text(headerBannerHint + " ") +
-        Text(Localization.headerBannerLearnMoreHint)
-            .font(POSFontStyle.posBodySmallBold.font())
+    private var bannerHintAndLearnMoreText: some View {
+        Text("\(headerBannerHint) \(Localization.headerBannerLearnMoreHint)")
+            .font(.posBodySmallBold)
```

Comment:
Should we use something like `String.localizedStringWithFormat( ... )` here rather than string concatenation? For localization in case of sites/app settings running RTL languages

Reply:
Good question. It also depends how we want to present this part. On Android, only "Learn more" is presented in primary color while on iOS we show these two sentences from a new line. Either way, learn more is looks to be designed to be shown in a new sentence.